packages/react-native-dogfood/types.ts

```diff
@@ -142,5 +142,4 @@ export type ScreenTypes =
   | 'lobby'
   | 'error-join'
   | 'error-leave'
-  | 'loading'
```

Comment:
I had to remove this state altogether because showing a loading state while joining breaks the reconnection flow.. since it will unmount the active call component